portal-impl/src/com/liferay/portal/service/impl/WorkflowInstanceLinkLocalServiceImpl.java

```diff
@@ -109,6 +111,27 @@ public WorkflowInstanceLink deleteWorkflowInstanceLink(
 		return workflowInstanceLink;
 	}
 
+	@Override
+	public void deleteWorkflowInstanceLinkByWorkflowInstanceId(
+			long workflowInstanceId)
+		throws PortalException {
+
+		WorkflowInstanceLink workflowInstanceLink =
+			workflowInstanceLinkPersistence.fetchByWorkflowInstanceId(
+				workflowInstanceId);
+
+		if (workflowInstanceLink == null) {
+			if (_log.isDebugEnabled()) {
+				_log.debug(workflowInstanceId);
```

Comment:
@mxs2 let's set good a message for this log:

`"Workflow instance link is already deleted with the workflowInstanceId: " + workflowInstanceId`